test/e2e/data/cluster-templates/docker-rke2.yaml

```diff
@@ -28,72 +16,152 @@ spec:
       cidrBlocks:
       - 10.10.0.0/16
     serviceDomain: cluster.local
-  controlPlaneRef:
-    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
-    kind: RKE2ControlPlane
-    name: ${CLUSTER_NAME}-control-plane
-  infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-    kind: DockerCluster
-    name: ${CLUSTER_NAME}
+  topology:
```

Comment:
Did you write this yourself or generate it using `clusterctl generate` or some other tool?

Reply:
I wrote it manually